src/main/java/Duke/Printer.java

```diff
@@ -0,0 +1,124 @@
+package Duke;
+
+public class Printer {
+    private static final String BORDER = "    ____________________________________________________________\n";
+    private static final String SPACING = "    ";
+    public static final String LOGO = " ____        _        \n"
+                                    + "|  _ \\ _   _| | _____ \n"
+                                    + "| | | | | | | |/ / _ \\\n"
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+
+    /**
+     * Prints default greeting from Duke.
+     */
+    public void printGreeting() {
+        System.out.println("Hello from\n" + LOGO);
+
+        System.out.println(BORDER + SPACING +
+                "Hello! I'm Duke\n" + SPACING +
+                "What can I do for you?\n" +
+                BORDER);
```

Comment:
Would it be better to add a line break before the `+` operator to improve readability? i.e. from

```java
System.out.println(BORDER + SPACING +
        "Hello! I'm Duke\n" + SPACING +
        "What can I do for you?\n" +
        BORDER);
```

to

```java
System.out.println(BORDER + SPACING
        + "Hello! I'm Duke\n" + SPACING
        + "What can I do for you?\n"
        + BORDER);
```